test_pytest.py

```diff
@@ -12,63 +11,78 @@ def test_client():
-    '''
     example_experience = {
-        "title": "Software Developer",
+        "role": "Software Developer",
         "company": "A Cooler Company",
-        "start_date": "October 2022",
-        "end_date": "Present",
+        "start": "October 2022",
+        "end": "Present",
         "description": "Writing JavaScript Code",
         "logo": "example-logo.png"
     }
 
-    item_id = app.test_client().post('/resume/experience',
-                                     json=example_experience).json['id']
-    response = app.test_client().get('/resume/experience')
-    assert response.json[item_id] == example_experience
+    post_response = client.post('/resume/experience', json=example_experience)
+    assert post_response.status_code == 201
+    item_index = post_response.json['index']
 
+    get_response = client.get(f'/resume/experience/{item_index}')
+    assert get_response.status_code == 200
+    data = get_response.json
 
-def test_education():
-    '''
-    Add a new education and then get all educations. 
-    
-    Check that it returns the new education in that list
-    '''
-    example_education = {
-        "course": "Engineering",
-        "school": "NYU",
-        "start_date": "October 2022",
-        "end_date": "August 2024",
-        "grade": "86%",
-        "logo": "example-logo.png"
-    }
-    item_id = app.test_client().post('/resume/education',
-                                     json=example_education).json['id']
+    assert data["role"] == example_experience["role"]
+    assert data["company"] == example_experience["company"]
+    assert data["start"] == example_experience["start"]
+    assert data["end"] == example_experience["end"]
+    assert data["description"] == example_experience["description"]
+    assert data["logo"] == example_experience["logo"]
 
-    response = app.test_client().get('/resume/education')
-    assert response.json[item_id] == example_education
+    post_response = client.post('/resume/experience', json=example_experience)
+    assert post_response.status_code == 201
+    item_index = post_response.json['index']
 
+    get_response = client.get('/resume/experience')
+    assert get_response.status_code == 200
+    experiences = get_response.json
 
-def test_skill():
-    '''
-    Add a new skill and then get all skills. 
-    
-    Check that it returns the new skill in that list
-    '''
-    example_skill = {
-        "name": "JavaScript",
-        "proficiency": "2-4 years",
-        "logo": "example-logo.png"
-    }
+    inserted_exp = experiences[item_index]
+    assert inserted_exp["role"] == example_experience["role"]
+    assert inserted_exp["company"] == example_experience["company"]
+    assert inserted_exp["start"] == example_experience["start"]
+    assert inserted_exp["end"] == example_experience["end"]
+    assert inserted_exp["description"] == example_experience["description"]
+    assert inserted_exp["logo"] == example_experience["logo"]
```

Comment:
please read this part:

```
'''
    Add a new skill and then get all skills. 
    
    Check that it returns the new skill in that list
'''
```
you need to make a POST request and then GET the skill back